spec/system/task_spec.rb

```diff
@@ -1,27 +1,30 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+
   describe 'Task一覧' do
+    let(:task) { create(:task, project_id: project.id) }
```

Comment:
associationを設定することで`project_id`の指定は不要になりますね！
```suggestion
    let(:task) { create(:task) }
```

Reply:
https://github.com/daigomiyoshi/rspec_app_exam/pull/1/commits/732c43e34d6d863292d4fa4d115253c8e040e3c0 にて`association`を追加して修正しました！
ただ一つ、そうすると`project.id`と`task.project_id`が一致しなくなってしまい…、最後の **it 'Taskが削除されること'** が通らなくなってしまいます。
いくつか調べましたが、うまく回避する方法が思いつかず、とりあえずid指定するままにしておりますが、もし何か良いやり方があればご教示いただいたけますと幸いです…！